reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -151,22 +250,29 @@ private synchronized void refreshAperture() {
 
         long now = Clock.now();
         boolean underRateLimit = now - lastApertureRefresh > APERTURE_REFRESH_PERIOD;
```

Comment:
This is named `underRateLimit` because this refresh is not to be rate limited because of the refresh period?